VMAccess/HandlerManifest.json

```diff
@@ -1,13 +1,14 @@
 [{
```

Comment:
Fixed.  jq converts 1.0 to 1.  I would rather keep the experience pure so we can just run the tool in the future without having to hand edit it afterwards.